traffic_ops/testing/compare/compare.go

```diff
@@ -32,9 +32,12 @@ import (
 
 	"github.com/apache/trafficcontrol/lib/go-tc"
 	"github.com/kelseyhightower/envconfig"
+	"github.com/pborman/getopt/v2"
```

Comment:
We're not using the long option processing anywhere else..  Isn't the standard "flag" package sufficient?

 If this is added,  the source must be included in the `vendor` directory,   license info properly documented in `LICENSE` and the appropriate lines added to `.dependency_license` and `.rat-excludes`.  Is it worth doing that?